package.json

```diff
@@ -7,11 +7,9 @@
   "private": true,
   "dependencies": {
     "@mattermost/compass-components": "^0.2.12",
-    "@mattermost/compass-icons": "^0.1.10",
+    "@mattermost/compass-icons": "^0.1.21",
```

Comment:
Have we checked this patch update does not cause any ui related changes :P

Reply:
I dont think so ... most of the time there are only icons added and not changed ... Although with v0.1.18 I added SVG components which make it easier to align the icons when not inline

Reply:
ok sounds good